news/2022-07-23-new-featured-artist-noisz.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "New Featured Artist: NOISZ"
+date: 2022-07-20 10:00:00 +0000
+---
+
+**NOISZ** expands our Featured Artist library with a whole bunch of original rhythm game compositions by artists like *Soleily*, *Nhato* and *Camellia*!
+
+![](https://assets.ppy.sh/artists/286/header.jpg)
+
+An unprecedented combo of the bullet hell genre, visual novels, and (most importantly) rhythm games, **NOISZ** is a video game backed by premier talents in the modern rhythm game universe. 
+
+As of today, **NOISZ**'s awesome soundtrack is now part of osu!.
+
+[**NOISZ**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/286)houses **11** songs (and their instrumental versions) created by this outstanding roster of musicians:
+
+- 3M-Project
```

Comment:
```suggestion
- M-Project
```

Reply:
should be `M-Project` instead of `3M-Project`

![image](https://user-images.githubusercontent.com/36564236/180607121-e1ca4a01-e06b-4c3e-9777-5eaf1cee44a3.png)

![image](https://user-images.githubusercontent.com/36564236/180607197-01a1e190-6be3-4f36-ab84-2334a8e0081b.png)